src/preamble.js

```diff
@@ -933,9 +943,9 @@ function updateGlobalBufferViews() {
   Module['HEAPF64'] = HEAPF64 = new Float64Array(buffer);
 }
 
-var STATIC_BASE = 0, STATICTOP = 0, staticSealed = false; // static area
```

Comment:
i asked this already but either it got lost or i missed the answer. why remove the initializations for all these?